react/dependency installer/e4e1b2e6-26a1-4b3b-b472-4464814fa69a.json

```diff
@@ -0,0 +1,51 @@
+{
+  "implementations": {
+    "es6-node": {
+      "inputFunctions": {
+        "045a62ad-7340-479a-b544-f10d56f16597": "const { exec } = require( \"child_process\" );\nconst { sync } = data;\nconst { dependencies } = params;\nif (data.start !== false ){\n  const command = \"cd \" + data.projectPath + \" && \" + \"npm install \" + params.dependencies.join(' ');\n  outputs.stdout( command, tag );\n  if (state.running){\n    outputs.bounced(data.sync, tag);\n  } else {\n    state.running = true;\n    exec(command, (error, stdout, stderr) => {\n      if (error) {\n        outputs.error(`${error}`, tag);\n      } else {\n        outputs.stdout(stdout || \"\", tag);\n        outputs.stderr(stderr || \"\", tag);\n        outputs.error(null, tag);\n      }\n      state.running = false;\n    });\n  }\n}"
+      }
+    }
+  },
+  "interface": {
+    "inputs": {
+      "93078915-107a-4854-baaf-e2cdd4c06760": {
+        "type": "string[]",
+        "name": "dependencies",
+        "description": "Array of npm dependencies to install.\n\neg.\n\n[\"a\",\"b\"]"
+      },
+      "045a62ad-7340-479a-b544-f10d56f16597": {
+        "type": "any",
+        "name": "sync",
+        "description": "{\n  \"projectPath\": string,\n  \"start\": boolean\n}"
+      }
+    },
+    "outputs": {
+      "86e1623a-d67c-4e74-98bb-449e29005d56": {
+        "type": "string",
+        "name": "stdout"
+      },
+      "735430f5-61ba-48c6-a8a6-e952edcb259e": {
+        "type": "string",
+        "name": "stderr"
+      },
+      "8b7a505c-873e-4936-9195-f1b3e28c5520": {
+        "type": "(string or null)",
+        "name": "error"
+      },
+      "4c3736af-6977-4632-b048-5446eaa87edd": {
+        "type": "any",
+        "name": "bounced"
+      }
+    }
+  },
+  "name": "react/Dependency Installer",
```

Comment:
If this is an internal node of  `react/React dependency installer` I suggest the conventional name of `react/React dependency installer/Internal`